scripts/backup/pg_backup.sh

```diff
@@ -82,3 +79,15 @@ else
 fi
 
 echo -e "\nAll database backups complete!"
+
+#####################################
+###### UPLOAD TO GOOGLE DRIVE #######
+#####################################
+
+
+echo -e "\n\\n Uploading backup to Google Drive"
+echo -e "--------------------------------------------\n"
+
+gdrive upload $FINAL_BACKUP_DIR"$DATABASE".sql.gz
```

Comment:
@carlosmiranda what is about timeouts? Do we need auto retrying for failed uploadings?

Reply:
@vladarefiev please see latest commit. I did not add auto-retry (I think this is out of scope here), but added an error message following the pattern for the other commands in the script.

Reply:
@carlosmiranda good, thanks. I would like to add `--timeout 0` too, what do you think?